Sources/SwiftDocC/Model/DocumentationNode.swift

```diff
@@ -335,12 +335,19 @@ public struct DocumentationNode {
     /// - Parameters:
     ///   - article: An optional documentation extension article.
     ///   - engine: A diagnostics engine.
-    mutating func initializeSymbolContent(documentationExtension: Article?, engine: DiagnosticEngine) {
+    mutating func initializeSymbolContent(
+        documentationExtension: Article?,
+        engine: DiagnosticEngine,
+        bundle: DocumentationBundle,
+        context: DocumentationContext
```

Comment:
We need to pass them all the way down to `DirectiveParser` because the `Metadata` initialiser needs it.

Reply:
Franklin and I talked about this some more offline. Ideally the directives wouldn't need the full context because it creates a cyclic dependency. They really only need some mechanism to load data but changing that is way out of scope for this PR.